modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/oauth/authcode/AuthorizationCodeOAuthHandler.java

```diff
@@ -78,6 +79,8 @@ public class AuthorizationCodeOAuthHandler extends OAuthHandler<AuthorizationCod
   // TODO: MULE-10837 this should be a plain old @Inject
   private LazyValue<HttpService> httpService;
 
+  private final boolean forceInvalidateStatusRetrieval = getClusterConfig().getClusterService().isEnabled();
```

Comment:
are you sure this is the proper thing to check? looking at the default impl, it always retunrs `true`.

Reply:
`getClusterConfig().getClusterService().isEnabled()` always returns true as you said regardless of cluster usage. Instead of that, we can check clusterId, that would be empty when we aren't using cluster.